src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -210,7 +210,7 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                             }
                         }
 
-                        await Task.Delay(1).ConfigureAwait(false);
+                        await Task.Yield();
```

Comment:
>  but Delay might not be, and likely isn't since the delay amount is so small?

afaik Task.Delay will always be asynchronous if the value is not exactly 0 (then it will just return a completed task).